rreplay.py

```diff
@@ -143,4 +153,9 @@ def main():
     os.unlink('rrdump_proc.pipe')
 
 if __name__ == '__main__':
-    main()
+    try:
```

Comment:
This is  a good idea.  We need to make sure that everything is rolled up correctly though.  This means correctly killing all of the processes that were generated up until the interrupt was caught.  Otherwise, they'll live on as zombies consuming resources.